src/typechecker.erl

```diff
@@ -594,9 +594,7 @@ normalize({user_type, P, Name, Args} = Type, TEnv) ->
                 {ok, T} ->
                     normalize(T, TEnv);
                 opaque ->
-                    Type;
-                not_found ->
```

Comment:
I still think this can happen, but we cannot test this, because gradualizer_db needs a module which doesn't compile, because it references undefined types internally. Rebar doesn't allow that, because it thinks all erl files under test/ are test suites.

As long as we want to handle errors which the compiler also catches, which we have talked about before, I think we should come up with a way to test this rather than removing the handling. The alternative is to run the compiler's checks whenever we parse erl files.

Reply:
How can we get a module that does not compile into gradualizer_db? If we can't do this via eunit/rebar3, a similar trick as for when testing the CLI could be used.

Reply:
`gradualizer -pa bad_module.erl -- module_calling_bad_module.erl`

Reply:
Sorry, `-pa` needs a directory, but you get the idea. (We could put bad modules under priv/test/ or so, to confuse rebar.)

But this doesn't need to be done now, does it? We can open an issue instead.

Reply:
-pa did not work - it needs a directory of beams. We do have `gradualizer_db:import_erl_files/1` though. By using that directly I was able to trigger the code I removed.